File: 40-model-routing/python-sdk/routing.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def get_routing_config():
    res = requests.get(f"{API}/model-routing/config",
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"🎯 Routing config: {res.json()['data']}")
    return res.json()["data"]

File: 39-model-routing/npm-package/routing.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function getRoutingConfig() {
  const res = await axios.get(`${API}/model-routing/config`,
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('🎯 Routing config:', res.data.data);
  return res.data.data;
}
